bHY6yh-d15355/cod-yyyiou_bHY6yh-d15355.xml

```diff
@@ -0,0 +1,153 @@
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 7</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+              <name ref="#alasdairwsliter">Alasdair W. Sliter</name>
+              <resp>Transcription Editor</resp>
+              <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 7</title>
+               <date when="2023-07-07">July 7, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-07-07" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#C" n="22-r"/>
+            <cb ed="#C" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d15355"><!-- p1l3-r7 -->
+            <head xml:id="bHY6yh-d15355-Hd1e108">Rubrica 7</head>
+            <head xml:id="bHY6yh-d15355-Hd1e111" type="question-title">De diuersitate moralitatum et naturalitatum diuersorum hominum</head>
+            <p xml:id="bHY6yh-d15355-d1e114">
+              De diuersitate mo<lb ed="#C" break="no"/>ralitatum 
+              et naturalitatum
+              <lb ed="#C"/>diuersorum hominum
+            </p>
+            <!--next paragraph begins on the second line of the previous paragraph-->
+            <p xml:id="bHY6yh-d15355-d1e121">
+              <ref xml:id="bHY6yh-d15355-Rd1e118">ARistoteles
+              <lb ed="#C"/>ethincorum
```

Comment:
If the text REALLy has "ethnicorum" it should be surrounded in the `<sic>` tag. but double check if it is better read as "ethicorum" referring to Aristotle's Ethics.
```suggestion
              <lb ed="#C"/>ethicorum
```